### ISSUE TITLE
Load on top instead of replacing

### ISSUE THREAD

**Issue by rcobiella** [gmrchk/swup#58]
Hi, and thanks for the great work!

I'm having problems with something I'm trying to do, and hope you can help. I have everything working fine and transitions look perfect, but there are some pages that I'd like to load on top as an overlay without replacing the previous Swup container but using a new one instead, and I can't seem to find the way.

This is a simplified structure for the website page:
Home
|__ Section 1
|__ Section 2
| ______ Subpage
| ______ Subpage
| ______ Subpage
|__ Section 3
|__ Section 4

Basically, I need to have two Swup instances (or any other possible workaround) with different LINK_SELECTORs, so Section 2 doesn't transition away and get replaced when I load one of its subpages.

Thanks in advance.

**Comment by gmrchk** [comment]
Hi @rcobiella!

Swup only works as an enhancement for your page, which kind of means that when you disable JavaScript, you should pretty much get the same pages. That would mean that your "Subpages" would need to have the "Section" page in a background. I'm guessing that's not the case, but that's just the main idea of swup and I'm sure we can figure out some workaround. 

swups [custom classes](https://github.com/gmrchk/swup#how-it-works) on `html` tag based on route or `data-swup-class` attribute could be helpful for modifying your animation for those transitions only.  

In case that you want to preserve the content of the "Section" page even on the "Subpage" pages (or for the animation at least), I can think of two options:

1. Before the transition to "Subpage" starts, copy the content of the "Section" page into some absolute/fixed position container that would be there only for the purpose of displaying the "Section" page during the animation. In general, swupjs could give you more control over what happends during animations and when are different animations supposed to happen.

2. Implement the opening of the "Subpage" page only as a modal, so swup wouldn't be really part of it, as there would be no routing. You could still use a fragment of URL (like `#this-subpage`) to open it on initial load of "Section" page and the "Section" page would always be on the background, even on the first load, which might save you some trouble for people that would come to the "Subpage" directly.

I never used more than one swup instance on the page, but I like the idea. Unfortunatelly, as swup is using classes on `html` tag for animations, it can't really have two instances working together. Swupjs, on the otherhand could do the trick in some way, but it wasn't intended to be used like that and some custom code would definitelly be needed (I don't recommend it). 

Cheers!

**Comment by rcobiella** [comment]
Thanks a lot for the reply.

I did find a solution that works for me switching from Swup to SwupJS, this way I can clone the page I want to keep before triggering the animation, pretty much what you said in 1.

I used to implement smoothState on my pages and then switched to Barba, but now I'm really glad I found Swup and I'm using it everywhere. It's just great!

**Comment by gmrchk** [comment]
Perfect! Glad you figured it out on your own, guess that's the most "fun" after all.

If you'd like to share your draft or final work, I would love to check it out. I'm curious about what people do with swup, so be sure to forward me a link if you feel like it. 

If you have some highlights of what you find nice about swup and why you switched to it, please mention that too for example [here](https://github.com/gmrchk/swup/issues/8). People have different preferences and it's nice to have a feedback of what are people looking for.

Cheers!

**Comment by niklasgrewe** [comment]
@rcobiella hi I am currently facing the same problem. Can you give me a small example of how you solved it?
